website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -62,6 +62,10 @@ default value in the `"/sys/config/ui"` [API endpoint](/vault/api-docs/system/co
   [go-sockaddr template](https://pkg.go.dev/github.com/hashicorp/go-sockaddr/template)
   that is resolved at runtime.
 
+- `chroot_namespace` `(string: "admin/")` – Specifies the namespace that will be used
+as the apparent root directory by this listener. When not specified, this will default to
+the usual "root/" namespace.
```

Comment:
```suggestion
the usual root namespace.
```

I'd rather not expose more people to this alias than we have to.  I might leave off the "when not specified" since I don't think it's necessary.